api/testdatajson.js

```diff
@@ -0,0 +1,13 @@
+const content = {
+      "title": "PCs don’t stand a chance…",
+      "transcript_by": "masterchief164 via TBTBTC v1.0.0\n",
+      "categories": "['hello', 'hi']",
+      "tags": "['tag1', 'tag2']",
+      "speakers": "['speaker1', 'speaker2']",
```

Comment:
having arrays is consistent with how it's done on the repo:

https://raw.githubusercontent.com/bitcointranscripts/bitcointranscripts/master/andreas-antonopoulos/2014-10-08-andreas-antonopolous-canada-senate-bitcoin.es.md